### PR TITLE
Description product

### PR DESCRIPTION
**Pull request by UrsuVasile**

### CODE REVIEW

File: src/main/java/Database/ProductDAO.java [hunk 1]
```diff
@@ -23,10 +23,11 @@ public void insertProduct(Product product) {
         closeSessionAndTransaction();
     }
 
-    public void updateProduct(Product product) {
+    public void updateProduct(Product product, Description description) {
         if (product.getId() != 0) {
             openSessionAndTransaction();
             session.update(product);
+            session.update(description);
```
Comment (stefan2212) [inline src/main/java/Database/ProductDAO.java]: delete this

**Review by stefan2212 — CHANGES_REQUESTED** [review]

**Review by andreihga — APPROVED** [review]

File: src/main/java/Database/ProductDAO.java (hunk 1 above)
Comment (UrsuVasile) [reply src/main/java/Database/ProductDAO.java]: deleted

**Review by andreihga — APPROVED** [review]

File: src/main/java/Database/ProductDAO.java [hunk 2]
```diff
@@ -23,7 +23,7 @@ public void insertProduct(Product product) {
         closeSessionAndTransaction();
     }
 
-    public void updateProduct(Product product) {
+    public void updateProduct(Product product, Description description) {
```
Comment (stefan2212) [inline src/main/java/Database/ProductDAO.java]: you don't need Description here

File: src/main/java/Service/Service.java [hunk 3]
```diff
@@ -121,10 +127,47 @@ public void executeProductCommand(int productCommand) {
...
+//                int newId = Optional.ofNullable(newProduct)
+//                            .map(Product::getDescription)
+//                            .map(Description::getId)
+//                            .orElse(0);
+                description.setId(newId);
+
+                product.setDescription(description);
+                description.setProduct(product);
+
+                productDAO.updateProduct(product, description);
```
Comment (stefan2212) [inline src/main/java/Service/Service.java]: call this method withouth description

**Review by stefan2212 — CHANGES_REQUESTED** [review]

File: src/main/java/Database/ProductDAO.java (hunk 2 above)
Comment (UrsuVasile) [reply src/main/java/Database/ProductDAO.java]: deleted

File: src/main/java/Service/Service.java (hunk 3 above)
Comment (UrsuVasile) [reply src/main/java/Service/Service.java]: ok

**Review by stefan2212 — APPROVED** [review]

**Pull request merged**